src/app/components/forms/post/post.ts

```diff
@@ -111,8 +111,8 @@ export class FormPost extends BaseForm<FormPostModel>
 	};
 
 	readonly YOUTUBE_URL_REGEX = /^(https?:\/\/)?(www\.)?(youtube\.com\/watch\?v=|youtu\.be\/)([a-zA-Z0-9_\-]{11})(&.+)*$/i;
-	readonly SKETCHFAB_URL_REGEX = /^(https:\/\/)?(www.)?sketchfab.com\/models\/[0-9a-f]{32}\/?$/i;
-	readonly SKETCHFAB_FIELD_REGEX = /^((https:\/\/)?(www.)?(sketchfab.com\/models\/[0-9a-f]{32}\/?))|([0-9a-f]{32})$/i;
+	readonly SKETCHFAB_URL_REGEX = /^((https:\/\/)?(www.)?(sketchfab.com\/3d-models\/([a-z0-9]+-)+[0-9a-f]{32}\/?))\/?$/i;
```

Comment:
Use non capturing groups for anything you dont care to match on. Make sure the model id is the only bit that has a capturing group. Something like `3d-models\/[\w\d\-]+?-([0-9a-f]{32})\/?` seems to work better.

Make sure to use that matcher instead of substring later on to extract the model ID.